docs/server/source/cloud-deployment-templates/node-on-kubernetes.rst

```diff
@@ -367,48 +367,111 @@ Create the required Deployment using:
 You can check its status using the command ``kubectl get deploy -w``
 
 
-Step 10: Verify the BigchainDB Node Setup
------------------------------------------
+Step 10: Run NGINX as a Deployment
+----------------------------------
 
-Step 10.1: Testing Externally
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+NGINX is used as a proxy to both the BigchainDB and MongoDB instances in the
+node.
+It proxies HTTP requests on port 80 to the BigchainDB backend, and TCP
+connections on port 27017 to the MongoDB backend.
 
-Try to access the ``<dns/ip of your exposed bigchaindb service endpoint>:9984``
-on your browser. You must receive a json output that shows the BigchainDB
-server version among other things.
+You can also configure a whitelist in NGINX to allow only connections from
+other instances in the MongoDB replica set to access the backend MongoDB
+instance.
 
-Try to access the ``<dns/ip of your exposed mongodb service endpoint>:27017``
-on your browser. You must receive a message from MongoDB stating that it
-doesn't allow HTTP connections to the port anymore.
+Get the file ``nginx-cm.yaml`` from GitHub using:
 
+.. code:: bash
+   
+   $ wget https://raw.githubusercontent.com/bigchaindb/bigchaindb/master/k8s/nginx/nginx-cm.yaml
 
-Step 10.2: Testing Internally
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The IP address whitelist can be explicitly configured in ``nginx-cm.yaml``
+file. You will need a list of the IP addresses of all the other MongoDB 
+instances in the cluster. If the MongoDB intances specify a hostname, then this
+needs to be resolved to the corresponding IP addresses. If the IP address of
```

Comment:
You could provide a bit more detail here. For example, maybe say that you have to edit the `nginx-cm.yaml` file and change the list of `allowed-hosts` to be a colon-separated list of IP addresses (of the other MongoDB instances in the BigchainDB cluster). A concrete example of a realistic-looking `  allowed-hosts:` line would go farther than 1000 words about general principles.